_summaries/park2019deepsdf_1.md

```diff
@@ -3,17 +3,48 @@ layout: summary
 title: Summary
 paper: {{paper_tag}}
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: patelojas
+score: 5
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+# Core Idea
+
+DeepSDF represents an approach to represent 3D geometryies for rendering and reconstruction and is a novel aproach to generative 3D modeling. Given a 2D or partial 3D observation, we wish to reconstruct a 3D model. It is used in robotics, simulations, etc. Other approaches face a similar trade-off across fidelity, efficiency, and compression capabilities. The goal is to maximize these traits or determine an appropriate balance between them. DeepSDF is a learned continuous Signed Distance Function that represents a class of shapes that enables high quality shape representation, interpolation, and completion from partial and noisy 3D input data. It represents a shape's surface by a continuous volumetric field. It uses the magnitude of a point in the field to represent the distance to the surface boundary and uses the sign to indicate whether the region is inside or outside of the shape. One key advantage of DeepSDF is that it allows the ability to finely represent surface details. 
+
+# Difference Between Classical SDF
+
+Classical SDF typically represents the surface of a single shape, but DeepSDF can represent an entire class of shapes. Previous methods include voxel (high memory requirement, point cloud (do not represent the surface) and mesh.
+
+# Technical Implementation
+
+DeepSDF is a continuous shape learning approach. The authors describe modeling shapes as the zero iso-surface decision boundaries of feed-forward networks trained to represent SDFs. They use a signed function, which is a continuous function that outputs a point's distance to the closest surface who's sign encodes whether the point is inside or outside of the surface. The surface is represented by SDF(*) = 0. This is used to predict the SDF value of a given query position to then extract the zero level-set surface. 
+
+$$ SDF(x) = s: x \in R^3, s \in R $$
+
+The most direct way to use this approach is to train a single deep network for a given shape. To then generate the 3D model, they use a feed-forward network composed of 8 fully connected layers with dropouts. 
+
+However, training a network for each shape is not practical. A better approach is to create a model that can represent a wide variety of shapes in a low dimensional latent space. 
```

Comment:
The technical implementation is a bit hand-wavy. How do they realize one network for a set of shapes? (Please check Sec. 4)